archetypes/data-playground.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here" 
+draft: true
+image: img/data-playground/
+link: "add github link"
+description: 'Add Description Here'
+summary: " Add summary here"
+license: "Add License Here"
+producer_name: "Add Producer name Here"
+producer_link: "Add Producer Link"
+api_key: "Add Api key Here"
```

Comment:
```suggestion
api_key: "Add API Key Here"
```

Reply:
This should probably be a `bool` - e.g. is api key required, true or false?